jdk7/src/main/java/pcap/jdk7/internal/NativeMappings.java

```diff
@@ -58,6 +61,16 @@ class NativeMappings {
   private static final Map<String, Object> NATIVE_LOAD_LIBRARY_OPTIONS =
       new HashMap<String, Object>();
 
+  static final File NPCAP_DIR = Paths.get(System.getenv("SystemRoot"), "System32", "Npcap").toFile();
```

Comment:
> and should be executed only once.

Was this in reference to moving `NPCAP_DIR` to the if block? Sorry, didn't quite understand what it means.

I looked into ensuring the path is only added once in case the static block runs again, but there doesn't seem to be a way to check that in JNA. That I could find anyways.